modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx

```diff
@@ -387,6 +388,19 @@ const AddFDSEntryModalContent = ({
 		}
 	};
 
+	const isValidPath = (path: string, allowedParameters: string[]):boolean => {
```

Comment:
(nit) when naming something, the most important word should be the last word in the name. It should determine what the function returns, or, in the the case of a boolean, if an adjective is true or false. Before that are descriptive words that determine scope. For example:
- getPath
- getValidPath
- getValidLocalPath
- isPathValid
- isLocalPathValid

Reply:
Thank you for the information, I'll change if for isPathValid